diagnostic.md

```diff
@@ -21,13 +21,24 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function (event) {
+  event.preventDefault();
+  donutsApi.index()
+    .then(logResponseBody)
+    .catch(logRequestError);
+  };
+const index = function () {
+  return $.ajax({
+    url: app.host + '/donuts',
+    method: 'GET',
+  });
+};
 ```
```

Comment:
what is donutsApi? Within the context of this question it has no value, we're looking for this:

```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```